res/xml/qwerty_vi.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key key0="q" key1="~" key2="1" key4="esc"/>
+    <key key0="w" key1="loc accent_breve" key2="2" key3="\@" key4="loc accent_horn"/>
+    <key key0="e" key1="!" key2="3" key3="\#" key4="ê"/>
+    <key key0="r" key2="4" key3="$" key1="loc accent_hook_above"/>
```

Comment:
All the accents that you need shouldn't have the `loc ` prefix. Like this, someone who has not the languages enabled on their device but decide to use this layout won't see the accents.

Reply:
removed in latest commit